sim/midas/targetutils/src/main/scala/midas/annotations.scala

```diff
@@ -220,31 +256,44 @@ object PerfCounter {
     *
     * @param label A verilog-friendly identifier for the event signal
     *
-    * @param message A description of the event.
+    * @param description A description of the event.
```

Comment:
Yeah, I was being lazy here, I'll quote the description strings.

Reply:
I think it is fine to be lazy and blow up if someone wants to have a comma in their description string... it would just be annoying if you allow commas to be there and not be quoted.